packages/node/src/breadcrumbs/FileBreadcrumbsStorage.ts

```diff
@@ -39,20 +40,13 @@ export class FileBreadcrumbsStorage implements BreadcrumbsStorage {
         );
     }
 
-    public static createFromSession(
-        session: SessionFiles,
-        fileSystem: NodeFileSystem,
-    ): FileBreadcrumbsStorage | undefined {
+    public static getSessionAttachments(session: SessionFiles) {
```

Comment:
I think we're breaking api here. Why we cannot continue using fileSystem?

Reply:
It's just not required anymore, as we're not creating `FileBreadcrumbsStorage`. Instead we just return attachments.